scripts/vm.sh

```diff
@@ -113,6 +105,58 @@ EOF
 	echo "Use: ssh $DEFAULT_USERNAME@$ip (password: $DEFAULT_PASSWORD)"
 }
 
+function is_gentoo() {
+	if uname -r | grep -iq "gentoo"; then
+		true
+	else
+		false
+	fi
+}
+
+function install_libs() {
+	if is_gentoo; then
+		echo "Gentoo system detected. Running emerge --sync..."
+		emerge --sync
+		emerge --deep app-emulation/libvirt app-emulation/qemu virtiofsd app-emulation/virt-manager whois
+		systemctl start libvirtd
+	else
+		echo "Non-Gentoo system detected. Running apt update..."
+		apt update
+		apt install -y libvirt-daemon-system virtiofsd
+	fi
+}
+
+function config_permissions() {
+	log "Configuring libvirt user permission and network"
+	if ! groups $USER | grep -q "\blibvirt\b"; then
+		usermod -a -G libvirt $USER
+	fi
+}
+
+function qemu_connect() {
```

Comment:
This function name does not describe what it does (all `virsh` commands connect to qemu in a way or another). I suggest `config_virsh_network`.
```suggestion
function config_virsh_network() {
```